src/components/10-atoms/input-text/README.md

```diff
@@ -146,6 +146,16 @@ The string-valued `info` provides the label text as HTML.
 
 _Note: The application is responsible for sanitizing HTML!_
 
+### pattern
+
+It is a string value. Useful to open up numeric keyboard on touch devices. See official HTML documentation for details.
+
+Default: ".\*" This RegEx allows everything. An empty string causes errors on form validation.
```

Comment:
I have no idea what this pattern property is about. It makes the numeric keyboard available, but what kind of regex is it that you are writing about here and what does "allows everything" mean? What is there to be allowed, if you just want to display a numeric keyboard?

Reply:
Re-write that